src/main/java/com/knu/buga1chuk/algo/search/BinarySearch.java

```diff
@@ -1,14 +1,17 @@
 package com.knu.buga1chuk.algo.search;
 
-public class BinerySearch {
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class BinarySearch {
+
+    private static final Logger LOG = LoggerFactory.getLogger(BinarySearch.class);
+
     /**
      * PrintSearchResult
      */
     public static void printSearchResult(int numberToFind, int idx) {
-        String messagePattern = "Element %d is located on index %d";
-        String result = String.format(messagePattern, numberToFind, idx);
-        System.out.println(result);
-
+        LOG.info("Element {} is located on index {}", numberToFind, idx);
```

Comment:
It is a good practice to wrap value. (with `'`, for example)
Like: `number: '{}'`

Reply:
done